docs/cpp/void-cpp.md

```diff
@@ -19,11 +19,17 @@ You can't declare a variable of type **`void`**.
 ## Example
 
 ```cpp
+void  print_num(int num)
+{
+   std::cout << num << std::endl;
+}
+
 // void.cpp
 void vobject;   // C2182
 void *pv;   // okay
 int *pint; int i;
-int main() {
+int main(void)
```

Comment:
I don't recommend this change. We should comment in the remarks that the use of `void` to denote an empty parameter list is a holdover from C and isn't recommended for C++ code. For more information, see [C++ Core Guidelines NL.25: Don't use `void` as an argument type](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl25-dont-use-void-as-an-argument-type).